std/process.d

```diff
@@ -2549,6 +2569,10 @@ private auto executeImpl(alias pipeFunc, Cmd, ExtraPipeFuncArgs...)(
     auto r3 = executeShell("exit 123");
     assert(r3.status == 123);
     assert(r3.output.empty);
+    auto r4 = executeShell("echo stderr test, please ignore 1>&2",
```

Comment:
Couldn't this be solved similar to #7641?

Reply:
Now you tell me... :-)

Done.